sdk/template/ci.yml

```diff
@@ -44,6 +44,8 @@ extends:
   parameters:
     ServiceDirectory: template
     EnableBatchRelease: true
+    # If you're copying template for a new pipeline, remove this variable. It is only valid for the template pipeline.
+    TestPipeline: true
```

Comment:
Not sure how but it would be nice if we could add some sort of chat that would fail if this is set for something that doesn't have "template" in the name.